### PR TITLE
Restyling 

### PR DESCRIPTION
**Pull request by greglobinski**
#### closed

Screenshot from 2019-04-08 18-22-07

#### opened
Screenshot from 2019-04-08 18-22-22

#### sending
Screenshot from 2019-04-08 18-22-58

#### success
Screenshot from 2019-04-08 18-24-19

#### failed
Screenshot from 2019-04-08 18-23-27

#### closed
Screenshot from 2019-04-08 18-32-10

#### opened
Screenshot from 2019-04-08 18-21-26

~~Note: @jlengstorf, I did not remove the code of the original `sending` micro animation (but commented it out) just in case you find the button animation is not enough.~~

### CODE REVIEW

File: src/components/feedback-widget/buttons.js [hunk 1]
```diff
@@ -1,40 +1,187 @@
...
+  }
 `
 
 export const SubmitButton = styled("button")`
   ${buttonStyles};
-  margin-right: 0.5rem;
+
+  .submitting & {
+    svg {
+      animation: ${rotation} 1s linear infinite;
```
Comment (marcysutton) [inline src/components/feedback-widget/buttons.js:60]: Can we add prefers-reduced-motion to the animations and transitions so people can turn it off?

Comment (greglobinski) [reply src/components/feedback-widget/buttons.js:60]: @marcysutton Sure, but do you mean, for all animations: opening widget, relocating toggle button, this small button icon animation ...?

Comment (marcysutton) [reply src/components/feedback-widget/buttons.js:60]: All animations would be ideal...but particularly the part where the widget opens and the close button slides up.

Comment (greglobinski) [reply src/components/feedback-widget/buttons.js:60]: @marcysutton 

I was too optimistic when I wrote "sure". :)

I'm on Ubuntu and Android and seems that this feature is not available on these systems. Correct me if I'm wrong. This is the first time I use it.

I added some `@media` queries but I'm not able to test if they work as they should. Could you please check it?

Comment (marcysutton) [reply src/components/feedback-widget/buttons.js:60]: Ah yes, reduced-motion will only work on supported platforms (OSX and maybe Windows), in supported browsers (Safari, Firefox and soon Chrome).

Comment (greglobinski) [reply src/components/feedback-widget/buttons.js:60]: @marcysutton OK, I turned off all UI animations (when `prefers-reduced-motion: reduce`) and tested it out on iOS11 device. The only left animations are these interaction animations of 'faces'. If you prefer I can turn them off also but IMHO they add some fun to the form and we should not hide them.

Comment (jlengstorf) [reply src/components/feedback-widget/buttons.js:60]: @marcysutton Any follow-up thoughts here? Or can we merge this?

Comment (marcysutton) [reply src/components/feedback-widget/buttons.js:60]: I think go ahead and merge it, and we can make any necessary changes later!

**Pull request merged**